CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
 # Contributing to Real-time Ubuntu
```

Comment:
I think for now it would be helpful to be clear that this repo is for contributing to docs; not the RTU kernel source.

`# Contributing to Real-time Ubuntu docs`.